lib/validate.ts

```diff
@@ -174,9 +174,13 @@ export async function validate(
     // parse everything for deeper testing
     let componentSet: ComponentSet | null = null;
     try {
-        componentSet = await parseDefinition(await loadHtmlRenditions(componentsDefinition, getFileContent));
+        componentSet = parseDefinition(await loadHtmlRenditions(componentsDefinition, getFileContent));
     } catch (e) {
-        errorReporter(e.message ?? e);
+        if (!(e instanceof Error)) {
+            // Re-throw any non Error objects as these should not happen (software error, not a component set issue)
+            throw e;
+        }
+        errorReporter(e.message);
```

Comment:
Had to make these changes as `parseDefinition` does not contain any async logic and we don't want to make the `processInfo` function async.

Reply:
hmm, I'm a bit unsure. Eventually we might add async logic to the parsing?

Reply:
It just feels weird to make these functions async when they not contain async logic. But from an "API" perspective allowing async logic in the future I guess it could make sense.